plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -49,6 +49,8 @@ export default function Echart({
       chartRef.current?.on(name, handler);
     });
 
+    chartRef.current.clear();
```

Comment:
It is a temporary solution for Echarts related issues https://github.com/apache/echarts/issues/14857 before I knew  [#1044](https://github.com/apache-superset/superset-ui/pull/1044).Here I will remove it and wait for Echarts v5.2.0 to solve this problem.

Reply:
> It is a temporary solution for Echarts related issues [apache/echarts#14857](https://github.com/apache/echarts/issues/14857) before I knew [#1044](https://github.com/apache-superset/superset-ui/pull/1044).Here I will remove it and wait for Echarts v5.2.0 to solve this problem.

Thanks for clarifying @stepehLYZ ! As this will cause unnecessary rerendering of all ECharts charts, could we add a new prop `forceClear` or similar to the `Echart` component that defaults to 'false'? Then we can use set that to `true` for `Treemap`, avoiding the rerendering problem in other charts.

Reply:
Thats a good suggestion 👍 .

Reply:
Done.